src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -727,5 +728,24 @@ internal static bool ConnectionExists(IEnumerable<KeyValuePair<string, string>>
                 : null;
             return (new Uri($"{protocol}://0.0.0.0:{Port}"), new Uri($"{protocol}://localhost:{Port}"), cert);
         }
+        private void EnsureWorkerRuntimeIsSet()
```

Comment:
new line between functions
```suggestion
        
        private void EnsureWorkerRuntimeIsSet()
```